rules/windows/persistence_ms_outlook_vba_template.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = "Detects attempts to establish persistence on an endpoint by installing a rogue Microsoft Outlook VBA Template."
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Persistence via Microsoft Outlook VBA"
+references = [
+    "https://www.mdsec.co.uk/2020/11/a-fresh-outlook-on-mail-based-persistence/",
+    "https://www.linkedin.com/pulse/outlook-backdoor-using-vba-samir-b-/",
+]
+risk_score = 43
+rule_id = "397945f3-d39a-4e6f-8bcb-9656c2031438"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+file where event.type != "deletion" and
+ wildcard(file.path, "C:\\Users\\*\\AppData\\Roaming\\Microsoft\\Outlook\\VbaProject.OTM") and
+ not wildcard(process.executable, "C:\\Program Files*\\Microsoft Office\\root\\Office*\\OUTLOOK.EXE")
```

Comment:
Curious on your thoughts to leaving this line out - which would detect if an attacker created the VBA from within outlook on the system. OTM files _should_ be rare enough that FP would be low - and you could add a snippet to `false_positives`.

Reply:
yep make sense (although doing it from within Outlook require interactive access to the machine like RDP)